src/test/java/nl/tudelft/ti2206/group9/level/StateTest.java

```diff
@@ -107,13 +108,13 @@ public void testSetPlayerName() {
 
     @Test
     public void testGetDefaultSaveDir() {
```

Comment:
This test should not be part of StateTest anymore, just like the Track distance tests.

<review, testing